vsintegration/tests/UnitTests/LegacyLanguageService/Tests.LanguageService.Completion.fs

```diff
@@ -3920,6 +3920,13 @@ let x = query { for bbbb in abbbbc(*D0*) do
             [ "Parallel" ] // should contain (module)
             [ "map" ] // should not contain (let-bound value)
 
+    [<Test>]
+    member public this.``OpenTypeNamespaceOrModule.CompletionOnlyContainsNamespaceOrModule.Case1``() =        
+        AssertAutoCompleteContains 
+            [ "open type System." ]
+            "." // marker
+            [ "Collections"; "Console"; "Int32" ] // should contain (namespace and two types)
```

Comment:
If possible, we should avoid adding to this test suite and use the Roslyn-based ones instead. These are going to be a lot harder to bring over to an LSP-ified environemtn :(